Userland/Utilities/readelf.cpp

```diff
@@ -242,6 +242,7 @@ int main(int argc, char** argv)
     static bool display_unwind_info = false;
     static bool display_dynamic_section = false;
     static bool display_hardening = false;
+    const char* string_dump_section = nullptr;
```

Comment:
`StringView` this pls, we're not living in caves and using C : )

Reply:
Fixed :^)